src/components/UpdateButton/UpdateVisualizationContainer.js

```diff
@@ -0,0 +1,68 @@
+import { connect } from 'react-redux'
+import { tSetCurrentFromUi } from '../../actions/current.js'
+import {
+    acSetLoadError,
+    acClearLoadError,
+    acSetVisualizationLoading,
+} from '../../actions/loader.js'
+import { acSetShowExpandedLayoutPanel } from '../../actions/ui.js'
+import { GenericClientError } from '../../modules/error.js'
+import { validateLayout } from '../../modules/layoutValidation.js'
+import { sGetCurrent, sGetCurrentFromUi } from '../../reducers/current.js'
+
+// TODO: Replace all 'Update' buttons throughout the app with UVC
```

Comment:
Is there any Update button not replaced left?

Reply:
Nope, comment removed 👍